### DISCUSSION TITLE
Retrospect - Fatal error each time plugin is run

### DISCUSSION THREAD

**Discussion by faidyll** [retrospect-addon/plugin.video.retrospect#1622] [Q&A]
Hoping for some help, please:

I am totally new to Kodi and the Retrospect plugin and became interested in these to simplify video streaming on a Fire TV Stick 4K Max for an elderly person. I spent a lot of time customising the plugin settings to my liking and then experimented with controlling the device using Vysor with the aim of providing instruction and support to the user from a remote location. 

During this process and without knowingly making any changes to the setup, an error condition was created which now produces the following message each time Retrospect is run:

**Retrospect error
Check the log for more information.**

The offending condition is perhaps that 'cookiejar.dat' is an empty file.

20220322 04:54:26 - [CRITICAL] - addon.py             - 70   - + http.cookiejar.LoadError: '/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/userdata/addon_data/plugin.video.retrospect/cookiejar.dat' does not look like a Netscape format cookies file

To recover from this problem, can I reinstall the plugin while preserving the custom settings I have created? Alternatively, is there  another approach I should take?

Thanks in advance for any advice offered.

[retrospect 20220322 0454.log](https://github.com/retrospect-addon/plugin.video.retrospect/files/8318668/retrospect.20220322.0454.log)
[kodi 2022-03-22 0454.log](https://github.com/retrospect-addon/plugin.video.retrospect/files/8318670/kodi.2022-03-22.0454.log)

**Comment by basrieter** [comment]
Delete file and try again.

**Reply by faidyll** [reply]
Thank you. Deleting 'cookiejar.dat' solved the issue.
